nexus/src/app/update.rs

```diff
@@ -626,23 +740,24 @@ mod tests {
         let nexus = &cptestctx.server.apictx.nexus;
         let opctx = test_opctx(&cptestctx);
 
-        // starts out empty
+        // starts out populated
         let components = nexus
             .updateable_components_list_by_id(&opctx, &test_pagparams())
             .await
             .unwrap();
 
-        assert_eq!(components.len(), 0);
+        assert_eq!(components.len(), 9);
 
         // with no components these should both 500. as discussed in the
         // implementation, this is appropriate because we should never be
         // running the external API without components populated
-        let low =
-            nexus.lowest_component_system_version(&opctx).await.unwrap_err();
-        assert_matches!(low, external::Error::InternalError { .. });
-        let high =
-            nexus.highest_component_system_version(&opctx).await.unwrap_err();
-        assert_matches!(high, external::Error::InternalError { .. });
+        //
+        // let low =
+        //     nexus.lowest_component_system_version(&opctx).await.unwrap_err();
+        // assert_matches!(low, external::Error::InternalError { .. });
+        // let high =
+        //     nexus.highest_component_system_version(&opctx).await.unwrap_err();
+        // assert_matches!(high, external::Error::InternalError { .. });
```

Comment:
Because these are populated at startup, there's no longer a chance for these to 500. Not sure how to/whether to test this anymore.